paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/sink/FlinkSinkBuilder.java

```diff
@@ -221,7 +222,7 @@ public FlinkSinkBuilder clusteringIfPossible(
     /** Build {@link DataStreamSink}. */
     public DataStreamSink<?> build() {
         input = trySortInput(input);
-        DataStream<InternalRow> input = MapToInternalRow.map(this.input, table.rowType());
+        DataStream<InternalRow> input = mapToInternalRow(this.input, table.rowType());
```

Comment:
Why introduce this?

Reply:
According to offline discussion, I'll submit another PR to modify the accessibility of these methods.